spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionController.java

```diff
@@ -213,6 +214,17 @@ public void cleanup(@PathVariable("id") long id) {
 		this.taskDeleteService.cleanupExecution(id);
 	}
 
+	/**
+	 * Stop a set of task executions.
+	 *
+	 * @param ids the ids of the {@link TaskExecution}s to stop
+	 */
+	@RequestMapping(value = "/stop/{id}", method = RequestMethod.POST)
+	@ResponseStatus(HttpStatus.OK)
+	public void stop(@PathVariable("id") Set<Long> ids) {
```

Comment:
I see that this would users to pass comma-separated list of task execution IDs and stop them (along with their child task executions), do we intend to expose this as an option to stop a set of executions (and their child executions)?

Reply:
Good question.
In this case users would submit a list of task executions they wish to stop.  We can think of it as a user clicking a set of `TaskExecution`s from the UI and requesting a stop.    

While a user could request a stop for both parent and child `TaskExecution`s, they would not need to specify the child IDs if the child `TaskExecution` contained a parent id,  in that the code would stop the parent then search for the child ids to stop.